src/api/middleware/validators/offer.js

```diff
@@ -0,0 +1,96 @@
+const { body } = require("express-validator");
+
+const { useExpressValidators } = require("../errorHandler");
+const ValidationReasons = require("./validationReasons");
+const { arrayValuesInRange } = require("./validatorUtils");
+const JobTypes = require("../../../models/JobTypes");
+const FieldTypes = require("../../../models/FieldTypes");
+const TechnologyTypes = require("../../../models/TechnologyTypes");
+
+const create = useExpressValidators([
+    body("title", ValidationReasons.DEFAULT)
+        .exists().withMessage(ValidationReasons.REQUIRED).bail()
+        .isString().withMessage(ValidationReasons.STRING)
+        .isLength({ max: 90 }).withMessage(ValidationReasons.TOO_LONG(90))
+        .trim(),
+
+    body("publishDate", ValidationReasons.DEFAULT)
```

Comment:
The meaning has probably been lost in translation but for me it was a "publish this at this date". However, this assumption can be changed. What do you think would be best?

Reply:
It was just to make sure it makes sense being there, with that goal, it can be there. Otherwise, it should be placed internally. Btw it should be optional, and if not provided, default to current time (immediate publish)

Reply:
Will mark it as optional in the validators (and tests) and add a default value to the service, to default to new Date(Date.now())

Reply:
Done!